rest_framework/urlpatterns.py

```diff
@@ -11,7 +11,7 @@ def _get_format_path_converter(suffix_kwarg, allowed):
         else:
             allowed_cases = "|".join(allowed)
             allowed_pattern = f'(?:{allowed_cases})'
-        suffix_pattern = r"\.%s/?" % allowed_pattern
+        suffix_pattern = fr"\.{allowed_pattern}/?"
```

Comment:
Not convinced I like the `fr`-strings.

Perhaps we should just leave those ones as they are for now?

Reply:
changed back in latest update